### DISCUSSION TITLE
Table problem 0.17.4 versus 0.19.0

### DISCUSSION THREAD

**Discussion by DrakMrak** [ant-design-blazor/ant-design-blazor#3879] [Q&A]
I have a project where basically everything works correctly built on Ant Design components version 0.17.4
I've updated to version 0.19.0 (I've tried other versions in between)
And I have a problem with the Table component
Loading the table ends with an error:

**Microsoft.JSInterop.JSException: can't access property "querySelectorAll", o.tHead is undefined** .....

I tried everything possible, including loading via OnAfterRenderAsync  and nothing works for this problem.
If I go back to version 0.17.4 everything is fine again

Thank you in advance for any suggestions on how to solve this.
Best regards DM

c#, blazor, server-side application

**Comment by DrakMrak** [comment]
Looks like I figured out the problem, interestingly I didn't have the problem on version 17.4.

**Reply by ElderJames** [reply]
Hello @DrakMrak , could you please provide a reproduce sample for the issue?

**Reply by DrakMrak** [reply]
I've already sorted it out the fault was mine after all. During the initial loading and handling of the **(Table)OnChange** event I was starting duplicate data loading... why it worked in version 17.4 I don't know but since version 18 I was looking for the error and finally the correct handling of the **(Table)OnChange Event** prevented the problems.  

I was looking for something else because it was generating an error about not loading the table header...
"Microsoft.JSInterop.JSException: can't access property "querySelectorAll", o.tHead is undefined"
But in the end the problem consisted in duplicate data retrieval from DB in one moment... and apparently something has changed since version 18 

I closed the comment today after finding a problem with myself so I apologize...

**Reply by ElderJames** [reply]
This may be related to an issue with resizable, but we've fixed it.
https://github.com/ant-design-blazor/ant-design-blazor/pull/3746

**Reply by DrakMrak** [reply]
I understand, however there was also my not quite good data retrieval from the database so I understand that it could have manifested itself after some correction. OK by me is done. 👍
